config.xml

```diff
@@ -88,4 +83,5 @@
     <plugin name="cordova-plugin-statusbar" spec="2.2.2" />
     <plugin name="cordova-plugin-device" spec="1.1.4" />
     <plugin name="cordova-plugin-splashscreen" spec="~4.0.1" />
+    <plugin name="cordova-plugin-geolocation" spec="^2.4.3" />
```

Comment:
We don not need this plugin at this moment. To remove it `ionic cordova plugin rm cordova-plugin-geolocation`

Reply:
done!